app/package.json

```diff
@@ -38,16 +38,18 @@
   },
   "scripts": {
     "start": "meteor",
-    "start-coverage": "COVERAGE=1 COVERAGE_VERBOSE=1 COVERAGE_APP_FOLDER=${PWD/#$HOME/~}/ meteor",
     "pretest": "echo 'npm run lint --silent'",
     "test": "meteor test --once --driver-package dispatch:mocha-phantomjs --port 3100",
     "test-app": "meteor test --full-app --once --driver-package dispatch:mocha-phantomjs --port 3100",
     "test-watch": "meteor test --driver-package practicalmeteor:mocha --port 3100",
     "test-app-watch": "meteor test --full-app --driver-package practicalmeteor:mocha --port 3100",
     "test-watch-terminal": "TEST_WATCH=1 meteor test --driver-package dispatch:mocha-phantomjs --port 3100",
     "test-app-watch-terminal": "TEST_WATCH=1 meteor test --full-app --driver-package dispatch:mocha-phantomjs --port 3100",
-    "test-coverage-app-unit": "node_modules/spacejam/bin/spacejam test            --driver-package practicalmeteor:mocha-console-runner --loglevel debug --coverage out_coverage",
-    "test-coverage-app-full": "node_modules/spacejam/bin/spacejam test --full-app --driver-package practicalmeteor:mocha-console-runner --loglevel debug --coverage 'in_coverage|out_lcovonly'",
+    "coverage": "meteor npm run coverage-app-unit && meteor npm run coverage-app-full",
+    "coverage-app-unit": "spacejam-mocha test --coverage 'out_lcovonly out_coverage out_html' --settings settings.coverage.json",
```

Comment:
Yes it's spacejam that provides tons of additionnal binaries. However, you should not rely on spacejam-mocha for now, it fails sometimes, mainly on windows. Just replace spacejam-mocha into spacejam --driver-package practicalmeteor:mocha-console-runner for now

I don't want to publish my fork of spacejam with a different name because I don't approve some choices done with that software and I don't want to maintain it like meteor-coverage. 
A workaround would be with a tag, but I hope one day that spacejam owners will merge my fork.